src/app/resourcebundles/data/creation/en.properties

```diff
@@ -173,6 +173,8 @@ frmelmnts.snav.shareViaLink=Shared via link
 frmelmnts.snav.allmycontents = All My Content
 frmelmnts.snav.flagReviewer = Flag Review
 frmelmnts.snav.Collaboratingon = Collaborations
+frmelmnts.snav.textbook = textbook
```

Comment:
Keep it consistent Make textbook to Textbook

Reply:
done